### ISSUE TITLE
Transcribe LATITUDE LONGITUDE SUBROUTINES (pp. 1128-1134)

### ISSUE THREAD

**Issue by thewonderidiot** [virtualagc/virtualagc#506]
Transcribe the scans for the LATITUDE LONGITUDE SUBROUTINES log section into its source file. This section ranges from pages 1128 to 1134.

This section has a corresponding log section in Luminary 99, from which it has been pre-populated. The two should be quite similar; look for any differences and apply them to the Luminary 69 version of the file.

Reduced quality scans can be found [here](http://www.ibiblio.org/apollo/ScansForConversion/Luminary069/). The original quality scans are available at [the Internet Archive page for Luminary 69](https://archive.org/details/luminary6900miti). Note that the page numbers specified on this issue correspond to those printed on the original pages, not necessarily those in the filenames of the archive.org images. Look for the start of the log section; it should be marked with a "USER'S PAGE NO. 1".

**Comment by thewonderidiot** [comment]
@thomaspaulin , assigning this to you. Heads up -- it's not quite ready to be transcribed yet, as Ron is still working on proofing the Luminary 99 comments which make up the base of this. I'll let you know when it's ready.

**Comment by thomaspaulin** [comment]
@thewonderidiot, that suits me. Thanks for the head's up!

**Comment by thewonderidiot** [comment]
@thomaspaulin, this is now ready for transcription whenever you are. :smile:

**Comment by thomaspaulin** [comment]
@thewonderidiot on page 1130 in the top comments type section ALT -- ALTITUDE appears to be at the same tabulation as ERASABLE INIT. REQ. This appears to be the same with Luminary 99 yet the template for Luminary 99 transcriptions have the ALT -- ALTITUDE line at the same tabulation as those that surround it instead of being at one/half less. Furthermore the LUNAFLAG=0 FOR EARTH,1 FOR MOON line is halfway between on the scans but is not in the transcriptions. Please advise.

I'm inclined to leave at the same level seeing as the other transcriptions do this and the team has reviewed those transcriptions already.

**Comment by thewonderidiot** [comment]
@thomaspaulin Go ahead and fix those. The tools we've developed so far correct for the textual content of the characters, but completely ignores whitespace. We're hoping to make a tool to help us fix whitespace in the future... but for now, that's the only thing we don't have a good method to proof. :smile:

**Comment by rburkey2005** [comment]
Yes, it's a process.  The proofing tool can't distinguish between "THIS IS A TEST" and "T    HI S    ATE T".  I fixed a few such whitespace issues whilst proofing the Luminary 99 text, but there were far more than I was able to deal with in the time I had allotted to the process.  

I would amend what Mike said, in that I feel uncomfortable trying to hold people to higher standards than I have for myself.  Also, I'm somewhat responsible, since so many of these whitespace issues stem back to my original Luminary 131 and Colossus 249 transcriptions, in which I "improved" the spelling, punctuation, and whitespace to make them look pretty, and am now "correcting" them back to be identical to the printouts.  Then, over the years, these things I did were endlessly duplicated into other AGC program versions.  My penance is that I now finding myself correcting the same "improvements" over and over again.

So to you I'd say to certainly fix any whitespace problems that jump out at you and are annoying or objectionable to you ... but keep in mind that if you feel like you're spending so much time on that that you don't think you're making progress on the code itself, do what you feel is most-efficient for your own time usage.
